src/client/debugger/jupyter/debugControllers.ts

```diff
@@ -57,13 +57,15 @@ export class RunByLineController implements IDebuggingDelegate {
     public continue(): void {
         if (typeof this.lastPausedThreadId !== 'number') {
             traceVerbose(`No paused thread, can't do RBL`);
+            this.stop();
             return;
         }
 
         void this.debugAdapter.stepIn(this.lastPausedThreadId);
     }
 
     public stop(): void {
+        void executeDebugPyFix(this.kernel);
```

Comment:
Since comments would be good here, didn't make sense why this is required

Reply:
I'm also curious about this, we execute this code when we start, why do we have to execute it when we stop?